test/integration/alias_execution_test.exs

```diff
@@ -48,7 +48,7 @@ defmodule Integration.AliasExecutionTest do
   test "alias expansion fails on infinite expansion", %{user: user} do
     send_message(user, "@bot: operable:alias new my-alias \"my-alias\"")
     response = send_message(user, "@bot: my-alias") |> Map.fetch!("response")
-    assert "@vanstee Alias expansion limit (5) exceeded starting with alias 'my-alias'." = response
+    assert_error_message_contains(response, "An error occurred. Alias expansion limit (5) exceeded starting with alias 'my-alias'.")
```

Comment:
You are gonna have some merge conflicts with the tests. I added a helper to simplify stuff a bit.